cluster-appliances/configure

```diff
@@ -39,29 +39,53 @@ write_file('cluster-appliances.rc', appliances_vars.join("\n"))
 RUBY
 }
 
+_enable_alces_flight_www() {
+    local role, certname
```

Comment:
Hmm, `,` isn't idiomatic. It might work, but I don't know! Could you remove it please?